matriculation-plugin/src/main/java/fi/otavanopisto/muikku/plugins/matriculation/MatriculationRESTService.java

```diff
@@ -0,0 +1,165 @@
+package fi.otavanopisto.muikku.plugins.matriculation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.inject.Inject;
+import javax.ws.rs.GET;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+
+import fi.otavanopisto.muikku.model.users.UserEntity;
+import fi.otavanopisto.muikku.plugins.matriculation.restmodel.MatriculationExamAttendance;
+import fi.otavanopisto.muikku.plugins.matriculation.restmodel.MatriculationExamEnrollment;
+import fi.otavanopisto.muikku.plugins.transcriptofrecords.EducationTypeMappingNotSetException;
+import fi.otavanopisto.muikku.plugins.transcriptofrecords.TranscriptOfRecordsController;
+import fi.otavanopisto.muikku.plugins.transcriptofrecords.VopsLister;
+import fi.otavanopisto.muikku.schooldata.MatriculationSchoolDataBridge;
+import fi.otavanopisto.muikku.schooldata.RestCatchSchoolDataExceptions;
+import fi.otavanopisto.muikku.schooldata.SchoolDataIdentifier;
+import fi.otavanopisto.muikku.schooldata.entity.MatriculationExam;
+import fi.otavanopisto.muikku.schooldata.entity.User;
+import fi.otavanopisto.muikku.schooldata.entity.UserAddress;
+import fi.otavanopisto.muikku.schooldata.entity.UserPhoneNumber;
+import fi.otavanopisto.muikku.users.UserController;
+import fi.otavanopisto.muikku.users.UserEmailEntityController;
+import fi.otavanopisto.muikku.users.UserEntityController;
+import fi.otavanopisto.security.rest.RESTPermit;
+
+@Path("/matriculation")
+@Produces("application/json")
+@RestCatchSchoolDataExceptions
+public class MatriculationRESTService {
+
+  @Inject
+  private MatriculationSchoolDataBridge matriculationSchoolDataBridge;
```

Comment:
SchoolDataBridge shouldn't be injected but accessed like other Bridges via the datasource selection.